README.Rmd

```diff
@@ -34,20 +34,47 @@ Thus, imposing a smoothness restriction with *M* = 0 implies that the counterfac
 The package may be installed by using the function `install_github()` from the `remotes` package:
 
 ```{r,eval = FALSE}
-## Installation
+install.packages('remotes')
+# install from github
+remotes::install_github("asheshrambachan/HonestDiD")
+
+```
+
+Your installation may fail if the C++ headers are not configured correctly and you may see something to this effect.
+
+```
+Caused by error: 
+! Could not find tools necessary to compile a package
+Call `pkgbuild::check_build_tools(debug = TRUE)` to diagnose the problem.
+
+``` 
+The underlying issue could look something like this 
+
+
+```
+pkgbuild::check_build_tools(debug = TRUE)
+/Library/Frameworks/R.framework/Versions/4.4-arm64/Resources/library/RcppEigen/include/Eigen/src/Core/util/Macros.h:679:10: fatal error: 'cmath' file not found
+  679 | #include <cmath>
+      |          ^~~~~~~
+1 error generated.
+make: *** [foo.o] Error 1
+
+```
+To resolve this issue you should consult the [rstan installation page](https://github.com/stan-dev/rstan/wiki/Configuring-C---Toolchain-for-Mac) which will configure the C++ toolchain properly. After modifying the `makevars` file be sure to restart your R session.
+
+If all else fails then you can simply turn off warnings which will install the package without configuring the C++ toolchain. 
```

Comment:
Why is there a downside to doing this? We should include a note explaining the gains from doing the above.